publisher/socials/twitter.py

```
@@ -0,0 +1,149 @@
+        last_tweet_id = None
+        
+        for content, photo_src in clean_tweets:
+            tmp_path = None
+            try:
+                media_ids = None
+                if photo_src:
+                    
+                    # Скачиваем изображение, если это URL
+                    if photo_src.startswith('http'):
+                        try:
+                            tmp = tempfile.NamedTemporaryFile(delete=False)
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)